app/controllers/cookies_preferences_controller.rb

```diff
@@ -2,16 +2,28 @@ class CookiesPreferencesController < ApplicationController
   before_action :set_previous_url_in_session
 
   def new
-    @cookies_preferences_form = CookiesPreferencesForm.new({ cookies_consent: cookies["consented-to-cookies"] })
+    # TODO: Remove "consented-to-cookies" check after Dec 2023 (6 months from this comment). That cookie
+    # won't be valid anymore.
+    cookies_analytics_consent =
+      if cookies["consented-to-analytics-cookies"] == "yes" || cookies["consented-to-cookies"] == "yes"
```

Comment:
The current existing cookies agreement will set the `consented-to-analytics-cookies` option.